pkg/server/documents_operations.go

```diff
@@ -165,3 +165,15 @@ func (s *ImmuServer) CollectionUpdate(ctx context.Context, req *documentschema.C
 	}
 	return resp, nil
 }
+
+func (s *ImmuServer) DocumentInsertMany(ctx context.Context, req *documentschema.DocumentBulkInsertRequest) (*documentschema.DocumentBulkInsertResponse, error) {
+	db, err := s.getDBFromCtx(ctx, "DocumentInsertMany")
+	if err != nil {
+		return nil, err
+	}
+	resp, err := db.BulkInsertDocuments(ctx, req)
```

Comment:
it might be simpler to use same naming in both layers? `DocumentInsertMany` in database package as well?

Reply:
`return db.BulkInsertDocuments(ctx, req)`